lms/templates/courseware/courses.html

```diff
@@ -71,7 +71,7 @@
             %endfor
           </ul>
 
-          <ul class="courses-listing">
+          <ul class="courses-listing courses-list">
```

Comment:
what was reason we needed to add courses-list class?

Reply:
This test case `common.test.acceptance.tests.lms.test_lms_course_discovery.CourseDiscoveryTest#test_search`was failing. So we have to change the `common.test.acceptance.pages.lms.discovery.CourseDiscoveryPage#result_items` function's look up for results. I have tagged you there.